api/models/Ticket.js

```diff
@@ -4,21 +4,30 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const ticketSchema = new Schema({
-	requestor: { type: Schema.ObjectId, ref: 'User', required: true },
+	requestorId: { type: Schema.ObjectId, ref: 'User', required: true },
 	mentor: { type: Schema.ObjectId, ref: 'User' },
+	contact: { type: String, required: false },
```

Comment:
I still think contact here implies a person's name, but it is their phone number. So contactInfo may be better? What do you think?